generatedocs/README.md

```diff
@@ -15,8 +15,23 @@ The reason for having an intermediate "programmatically readable" step is to all
 
 ## How: Generating the documentation
 
+To generate new `documentation.go` and `settingup.md` files after updating the documentation data, use the following command in the project root:
+
+```bash
+go generate
+```
+
+The Gopherciser `main.go` file contains the following commands for `go generate`:
+
+```
+//go:generate go run ./generatedocs/compile/compile.go
+//go:generate go run ./generatedocs/generate/generate.go --output ./docs/settingup.md
+```
+
 ### Compiling documentation data to be used by the GUI and for markdown generation
 
+To generate a new `documentation.go` only use the following command:
```

Comment:
Suggestion:
"To only generate a new `documentation.go` file, use the following command:"